resources/integration_test/testdata/adl_db_config.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "name" : "integration_test",
+    "collections" : [
+      {
+        "name": "*",
+        "dataSources" : [
+          {
+            "storeName" : "localmongo",
+            "database" : "integration_test"
+          }
+        ]
+      }
+    ],
+    "views" : null
```

Comment:
Any reason why you didn't leave the same view config than the one on JDBC?
ODBC also has a concept of Table type and filtering, so I think it would make sense to add it.

Reply:
No strong reason, I wasn't sure if it would be used.  I've re-added it and we can test the table type and filtering as you mention.